rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -138,6 +138,11 @@ def botched_migration(cursor, rotki) -> None:
         'rotkehlchen.data_migrations.manager.MIGRATION_LIST',
         new=botched_list,
     )
+
+    # Ignore websocket messages with notifications about db upgrade version.
+    # By default they will be treated as errors since we have no websocket connection set up.
+    rotki.msg_aggregator.consume_errors()
+
```

Comment:
I'm not sure if I follow. If the type is in `INFORMATIONAL_MESSAGE_TYPES` they are not treated as errors and just ignored

Reply:
No since `failure_callback` is called, not the other part https://github.com/rotki/rotki/blob/be0400a7ca73f0684e043f7c10ac7a005d1775a9/rotkehlchen/user_messages.py#L82

Reply:
I also don't 100% get the problem here. But I guess has something to do with your new websocket messages?

Reply:
Yes. When messages aggregator fails to send a message through websockets (which is always true for tests) it calls this `failure_callback` which adds an error